how-to-build.md

```diff
@@ -64,11 +68,17 @@ Run the following commands to configure, build and test core:
     cmake --build .
     ctest
 
+If you wan to build the debug version use:
+
+    cmake --build -D CMAKE_BUILD_TYPE=Debug .
```

Comment:
This is not correct. The definition of CMAKE_BUILD_TYPE must happen on the first cmake invocation, the one that generates the project, not the one that builds it.

Reply:
Ok - can you fix it so it correct @emanuelez ?